### DISCUSSION TITLE
Other places?

### DISCUSSION THREAD

**Discussion by RyanWoo-Ming** [ruffle-rs/ruffle#4769] [Q&A]
After support for Adobe Flash Player dropped in the beginning of this year, I've been looking for places to play my swfs. I was going to use Ruffle, but after learning about its bugs, I don't even want to touch it. I also don't want to use other emulators because they might not work. Is it ok if I use something like the Homestar Runner Greasemonkey add-on or the Newgrounds Player, or just painstakingly edit the official player's files so that it can play swfs again?

**Comment by Bale001** [comment]
Ruffle actually has really good support for AS2 swfs, so if your swf's are written in AS2, Ruffle will probably work fine

**Comment by danielhjacobs** [comment]
Ruffle does not have many bugs really, it mostly has unimplemented features. If your swf doesn't use any of the unimplemented features, it usually works pretty well. I use it on some websites I maintain and it works pretty well. I'd try it before dismissing it.

**Comment by Poopooracoocoo** [comment]
>or just painstakingly edit the official player's files so that it can play swfs again?

look at flashpoint's modified flash player.
